### PR TITLE
Home page after logging in

### PR DESCRIPTION
**Pull request by sonlinux**
fix kartoza/LEDET_BIMS#69

## Left nav
left_navbar

## Right nav
right_navbar

### CODE REVIEW

File: bims/templates/main_base.html [hunk 1]
```diff
@@ -67,8 +67,11 @@
                      {% if user.is_authenticated %}
                         <li><a class="nav-link js-scroll-trigger" id="navbar-profile" href="/profile/">PROFILE</a></li>
                          <li><a class="nav-link js-scroll-trigger" id="navbar-contributions" href="/under-development/">CONTRIBUTIONS</a></li>
-                         <li><a class="nav-link js-scroll-trigger" id="navbar-upload" href="/upload/">UPLOAD FILES</a></li>
-                        <li><a class="nav-link js-scroll-trigger" href="/accounts/logout/"><i class="fa fa-sign-out"></i> LOGOUT</a></li>
+                         <li><a class="nav-link js-scroll-trigger" id="navbar-upload" href="/upload/">UPLOAD DATA</a></li>
+                         {% if user.is_staff or user.is_superuser %}
+                            <li><a class="nav-link js-scroll-trigger" id="navbar-upload" href="/admin/">ADMINISTRATION</a></li>
```
Comment (dimasciput) [inline bims/templates/main_base.html]: duplicate id for navbar-upload

Comment (sonlinux) [reply bims/templates/main_base.html]: Cleaned

**Comment by sonlinux** [comment]
## Remove contributions and confirm only admin can see Administration link
admin_not_admin

**Comment by sonlinux** [comment]
Done @gubuntu, @dimasciput 
Thanks

**Pull request merged**